index.html

```diff
@@ -0,0 +1,275 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tasnimul Hasan Portfolio</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+    <style>
+    </style>
+</head>
+<body>
+
+    <header>
+        <div class="nav-menu-content">
+
+            <div class="logo"><a class=mylogo> My Logo </a></div>
+
+            <nav>
+                <ul class="nav-menu">
+                    <li class="nav-items"><a href="#">Portfolio</a></li>
+                    <li class="nav-items"><a href="#">About</a></li>
+                    <li class="nav-items"><a href="#">Contact</a></li>
+                  </ul>
+
+
+                    <div class="hamburger">
+
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                    </div>
+            </nav>
+        </div>
+
+    </header>
+    <section class="homepage">
+        <div class="introduction">
+            <div class="intro"><h1>I'm Tasnimul <br>Glad to meet you!</h1></div>
+            <div class="me">
+                <p>
+                    I am a software developer. I have done some interesting projects using
+                    react js, node js and ruby on rails. I am learning new things and
+                    implementing them instantly. I can help you build a project, feature
+                    or website look through some of work and experience if you like what
+                    you see and have a project you need coded, don't hesitate to contact
+                    me.
+                </p>
+            </div>
+            <div class="connect">
+                <h3><a> Let's connect </a> </h3>
+            </div>
+            <div class="social">
+                <ul>
+                    <li class="icon"><a href=""><img src="Image/twitter.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/github.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/medium.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/angellist.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/Linkedinicon.png" alt="" /></a></li>
+                </ul>
+
+            </div>
+        </div>
+    </section>
+
+    <section class="portfolio-1">
+        <div class="project-1">
```

Comment:
- [ ] Your current work section is not matched with the Figma design. It's also not aligned properly. Please work on that. :+1: 

- [ ]  Kindly use the grid for the work section on the page. According to project requirements, you should use CSS Grid in the main container that contains all of the cards and use Flexbox to layout an individual card. :+1: 

> - **Your Implementation:**
![w1](https://user-images.githubusercontent.com/70483835/154630648-2a3176f3-851b-454c-a4e5-ca8cb1cba8b3.png)
> - **Figma Design:**
![w2](https://user-images.githubusercontent.com/70483835/154630820-dcaa99b3-c104-4f0a-9e0a-f42f571e25c0.png)

Reply:
Well done implementing the `display: grid` on this section 🚀